README.md

```diff
@@ -223,4 +221,4 @@ If you have questions or comments, please create an issue.
 
 ## License
 
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```

Comment:
This can be kept and added back in.

Reply:
done https://github.com/freeCodeCamp/how-to-contribute-to-open-source/commit/cf4b2cc817a90fd08f596925eecea5e29f070bda